### PR TITLE
Add more codeowners for Azure Cosmos DB

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

**Comment by Pilchie** [comment]
Also tagging named people as FYI: @kirankumarkolli @tvaron3 @FabianMeiswinkel @kundadebdatta

**Review by RickWinter — APPROVED** [review]

**Comment by ronniegeraghty** [comment]
Don't merge yet. Give me a few to test the aliases for the proper permissions since we don't have the codeowners-linter running in this repo yet

File: .github/CODEOWNERS
Comment (ronniegeraghty) [inline .github/CODEOWNERS:38]: For aliases to be valid in our `CODEOWNERS` file they need to have the following: 
- They need to be **PUBLIC** members of the Azure and microsoft GitHub Orgs
- They need to have write permission to the Azure SDK repos

❌@analogrelay, @kirankumarkolli and @FabianMeiswinkel are missing **public** membership to the microsoft Org.
❌@analogrelay and @tvaron3 are missing **public** membership to the Azure Org

✅@kundadebdatta and @Pilchie have all necessary permissions. 

Please view our [onboarding docs for getting access](https://aka.ms/azsdk/access) and go through the steps in the `Request access to Azure REST and SDK repositories` section. 

You can use the [`Validate-AzsdkCodeOwner.ps1`](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from our tools repo to test if your alias has all the required permissions. 
Ex: 
image
image

**Review by heaths — APPROVED** [review]

**Review by ronniegeraghty — DISMISSED** [review]

**Comment by heaths** [comment]
> Don't merge yet. Give me a few to test the aliases for the proper permissions since we don't have the codeowners-linter running in this repo yet

@ronniegeraghty and as such, the bot that uses the `Do Not Merge` label isn't running either so I recommend requesting changes on this PR to avoid it getting accidentally merged. 😉

File: .github/CODEOWNERS
Comment (kirankumarkolli) [reply .github/CODEOWNERS:38]: I did make my profile public fir both MS and Azure. 
I don't have access right to the tools repository.

Comment (analogrelay) [reply .github/CODEOWNERS:38]: I've updated my memberships to be public and am now showing as valid in the script✅. @kirankumarkolli I also see you are now valid, according to the script.

image

**Review by analogrelay — APPROVED** [review]
Looks good to me, once we've got all our owners compliant with the requirements 👍🏻

File: .github/CODEOWNERS
Comment (kirankumarkolli) [reply .github/CODEOWNERS:38]: Thank you @analogrelay!

Comment (kirankumarkolli) [reply .github/CODEOWNERS:38]: FYR Fabian is OOF right now.

Comment (ronniegeraghty) [reply .github/CODEOWNERS:38]: > I did make my profile public fir both MS and Azure. I don't have access right to the tools repository.

According to the script, your membership to the `microsoft` GitHub Org isn't public: 
image

Comment (analogrelay) [reply .github/CODEOWNERS:38]: @ronniegeraghty I see Kiran as valid now (see above, and I just ran the script again to confirm the same result). Are you still seeing that it's showing as invalid?

@kirankumarkolli I think we can wait for Fabian to return, so he can fix his membership visibility, before merging this. The PR volume in the Cosmos SDK is pretty low (it's basically just me right now 😅 ) and I can manually add reviewers to my PRs for now.

Comment (ronniegeraghty) [reply .github/CODEOWNERS:38]: > @ronniegeraghty I see Kiran as valid now (see above, and I just ran the script again to confirm the same result). Are you still seeing that it's showing as invalid?

I can confirm that Kiran's permission are showing up correctly now: 
image

Comment (heaths) [reply .github/CODEOWNERS:38]: If all is good, can you approve the PR and merge?

**Pull request merged**
